drizzle-kit/src/serializer/sqliteSerializer.ts

```diff
@@ -864,7 +864,7 @@ WHERE
 	const checkConstraints: Record<string, CheckConstraint> = {};
 	const checks = await db.query<{ tableName: string; sql: string }>(`SELECT name as "tableName", sql as "sql"
 		FROM sqlite_master 
-		WHERE type = 'table' AND name != 'sqlite_sequence';`);
+		WHERE type = 'table' AND name != 'sqlite_sequence' and name != '_cf_KV';`);
```

Comment:
it turns out that there is also an internal cloudflare table called [`_cf_METADATA`](https://github.com/cloudflare/workerd/blob/main/src/workerd/util/sqlite-metadata.h), and that, according to the comments in the related [`workerd`](https://github.com/cloudflare/workerd/blob/main/src/workerd/util/sqlite-kv.h#L28-L29) [classes](https://github.com/cloudflare/workerd/blob/main/src/workerd/util/sqlite-metadata.h#L19-L20), workerd “block[s apps] from accessing any table prefixed with `_cf_`”. so this should be something like:
```suggestion
		WHERE type = 'table' AND name != 'sqlite_sequence' AND name NOT LIKE '\_cf\_%' ESCAPE '\'
```